azurecontainerapps/azurecontainerapps.ts

```diff
@@ -88,6 +88,7 @@ export class azurecontainerapps {
     private static ingressEnabled: boolean;
 
     // ACR properties
```

Comment:
Just wanted to be 100% sure. Could you please confirm that the TypeScript changes are exactly the same as the GHA?

Reply:
The changes here has some slight difference (e.g. the way ADO task calls cli is a bit different from GHA), but after testing locally and e2e, i can confirm it works fine now.

Reply:
@harryli0108 curious how they are different? Ideally the flow should be the same for ADO and GitHub Action?

Reply:
GHA has this wrapper around each az cli calls in the file `GithubActionsToolHelper.ts`, where ado tasks (both main and rc) didn't have that. Other than that, the changes I made is basically the same as I did in GHA repo

Reply:
Ok, thanks for clarifying!